devsetup/scripts/standalone.sh

```diff
@@ -31,6 +31,7 @@ if [[ ${STANDALONE_VM} == "true" ]]; then
 fi
 IP_ADRESS_SUFFIX=${IP_ADRESS_SUFFIX:-"$((100+${EDPM_COMPUTE_SUFFIX}))"}
 IP=${IP:-"${EDPM_COMPUTE_NETWORK_IP%.*}.${IP_ADRESS_SUFFIX}"}
+CTLPLANE_IP=${CTLPLANE_IP:-"${IP}"}
```

Comment:
why do you need this override. I mean, why can't you use and set "IP" to the value that you want. the non standard ctlplane network.